src/main/js/main_html5.js

```diff
@@ -1163,7 +1163,12 @@ require("../../../html5-common/js/utils/environment.js");
     var convertToSafeSeekTime = function(time, duration) {
       // If seeking within some threshold of the end of the stream, seek to end of stream directly
       if (duration - time < OO.CONSTANTS.SEEK_TO_END_LIMIT) {
-        time = duration;
+        // iOS has a bug when seeking to the exact end of a video, the player will lock up and become unresponsive.
+        if (OO.isIos) {
```

Comment:
I do not think that this is the most efficient way on fixing it. If you think about it, Without this new code addition, duration will be subtracted by 0.01 in line 1174. But issue still persists.

You also do not need () in line 1168.